lib/squadster/schedules/services/create_timetable.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Squadster.Schedules.Services.CreateTimetable do
+  alias Squadster.Repo
+  alias Squadster.Formations.{Squad}
+  alias Squadster.Schedules.Timetable
+
+  def call(args, user, squad) do
+    squad
+    |> Repo.preload(:timetables)
+    |> case do
+      %{timetables: []} -> create(args, user)
+      %{timetables: _timetables} -> {:error, "Squad already has timetable"}
```

Comment:
Seems like it can only create one timetable for squad, is it correct?

Reply:
Sure, it's legacy stuff. I'll get rid of it